Sources/LiveViewNative/BuiltinRegistry.swift

```diff
@@ -122,4 +123,90 @@ struct BuiltinRegistry: BuiltinRegistryProtocol {
             try TintModifier(from: decoder)
         }
     }
+    
+    @ViewBuilder
+    static func applyBinding<R: CustomRegistry>(
+        _ binding: AttributeName,
+        event: String,
+        value: Payload,
+        to view: some View,
+        element: ElementNode,
+        context: LiveContext<R>
+    ) -> some View {
+        switch binding {
+        case "phx-window-focus":
+            ScenePhaseObserver(content: view) { scenePhase in
+                if scenePhase == .active {
+                    Task {
+                        try await context.coordinator.pushEvent(type: "focus", event: event, value: value)
+                    }
+                }
+            }
+        case "phx-window-blur":
+            ScenePhaseObserver(content: view) { scenePhase in
+                if scenePhase == .background {
+                    Task {
+                        try await context.coordinator.pushEvent(type: "blur", event: event, value: value)
+                    }
+                }
+            }
+        case "phx-focus":
+            FocusObserver(content: view) { isFocused in
+                guard isFocused else { return }
+                Task {
+                    try await context.coordinator.pushEvent(type: "focus", event: event, value: value)
+                }
+            }
+        case "phx-blur":
+            // Special case for `TextFieldProtocol`, which handles this event itself.
+            if element.tag == "text-field" || element.tag == "secure-field" {
```

Comment:
Another special case is needed for `phx-focus`/`phx-blur`, so this may be fairly common.